channel-email/src/main/java/com/synopsys/integration/alert/channel/email/convert/EmailGlobalFieldModelConverter.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * channel-email
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.channel.email.convert;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.action.api.GlobalFieldModelToConcreteConverter;
+import com.synopsys.integration.alert.common.rest.model.FieldModel;
+import com.synopsys.integration.alert.common.rest.model.FieldValueModel;
+import com.synopsys.integration.alert.service.email.model.EmailGlobalConfigModel;
+
+@Component
+public class EmailGlobalFieldModelConverter implements GlobalFieldModelToConcreteConverter<EmailGlobalConfigModel> {
+    private Logger logger = LoggerFactory.getLogger(getClass());
+    private static final String EMAIL_FIELD_PREFIX = "mail.smtp.";
+    public static final String AUTH_REQUIRED_KEY = EMAIL_FIELD_PREFIX + "auth";
+    public static final String EMAIL_FROM_KEY = EMAIL_FIELD_PREFIX + "from";
+    public static final String EMAIL_HOST_KEY = EMAIL_FIELD_PREFIX + "host";
+    public static final String AUTH_PASSWORD_KEY = EMAIL_FIELD_PREFIX + "password";
+    public static final String EMAIL_PORT_KEY = EMAIL_FIELD_PREFIX + "port";
+    public static final String AUTH_USER_KEY = EMAIL_FIELD_PREFIX + "user";
```

Comment:
Shouldn't these keys be available in EmailPropertyKeys? Is that not reachable from this class?

Reply:
yes we should be able to use this instead.